♻️ Moved short title generation in `event` module

ororktn opened the pull request.

Commit:
♻️ Moved short title generation in `event` module

Commit:
♻️ Moved comma-separated author string generator in `event.rs`

Lymkwi commented on src/event.rs:
```diff
@@ -75,6 +75,46 @@ pub struct Event {
+    if text.len() <= length {
+        text.to_owned()
+    } else {
+        let (short, _) = text.split_at(length - 3);
```
There is no check here that `length > 2`

Lymkwi reviewed (commented):
Nice

hreymond reviewed (approved):
Sounds ok to me. We can merge once @Lymkwi remark is addressed !

Commit:
🐛 Fixed potential panics in `event::cut_text`

- Can now handle `length` being lower than 3
- Now handling actual character count instead of byte-size
